prdoc/pr_3395.prdoc

```diff
@@ -0,0 +1,13 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: `benchmarking-cli`: refactor `list` and add `--all` option
+
+doc:
+  - audience: Node Dev
+    description: |
+      Makes `--list` accept two values: "all" and "pallets". The former will list all available benchmarks, the latter will list only pallets. 
```

Comment:
Can you still mention what sub-command this applies to please?  
Maybe also in the title.